rosdep/base.yaml

```
@@ -8255,6 +8255,14 @@ rapidjson-dev:
   nixos: [rapidjson]
   rhel: [rapidjson-devel]
   ubuntu: [rapidjson-dev]
+rclone:
+  ubuntu: [rclone]
+  debian: [rclone]
+  arch: [rclone]
+  alpine: [rclone]
+  rhel: [rclone]
+  fedora: [rclone]
+  opensuse: [rclone]
```

These have to be in alphabetical order:

```suggestion
rclone:
  alpine: [rclone]
  arch: [rclone]
  debian: [rclone]
  fedora: [rclone]
  opensuse: [rclone]
  rhel: [rclone]
  ubuntu: [rclone]
```